jdaviz/configs/mosviz/plugins/slit_overlay/slit_overlay.py

```diff
@@ -17,8 +17,8 @@
 def jwst_header_to_skyregion(header):
     s_region = header['S_REGION']
     footprint = s_region.split("POLYGON ICRS")[1].split()
-    ra = np.array(footprint[::2], dtype=np.float)
-    dec = np.array(footprint[1::2], dtype=np.float)
+    ra = np.array(footprint[::2], dtype=np.float_)
+    dec = np.array(footprint[1::2], dtype=np.float_)
```

Comment:
Isn't this what numpy actually suggested?

```suggestion
    ra = np.array(footprint[::2], dtype=float)
    dec = np.array(footprint[1::2], dtype=float)
```